src/js/load-package.ts

```diff
@@ -483,8 +502,10 @@ export async function loadPackage(
     );
 
     Module.reportUndefinedSymbols();
-    if (loaded.size > 0) {
-      const successNames = Array.from(loaded).join(", ");
+    if (loadedPackageData.size > 0) {
+      const successNames = Array.from(loadedPackageData)
+        .map((pkg) => pkg.name)
+        .join(", ");
```

Comment:
`Array.from(x).map(f)` should be written as `Array.from(x, f)`. And maybe we should sort these? 
```suggestion
      const successNames = Array.from(loadedPackageData, (pkg) => pkg.name).sort().join(", ");
```